docs/reference/api/pages.md

```diff
@@ -13,6 +15,18 @@ All [page types](TODO) use a single set of API endpoints, unlike piece types. Di
 |PATCH | [`/api/v1/@apostrophecms/page/:id`](#patch-api-v1-apostrophecms-page-id) | Update only certain fields on a specific page | TRUE |
 |DELETE | Not supported | [See more in PATCH request details.](#moving-pages-to-the-trash) | n/a |
 
+### Additional page endpoints
+
+| Method | Path | Description | Auth required |
+|---------|---------|---------|---------|
+|POST | [`/api/v1/@apostrophecms/page/:id/publish`](#post-api-v1-apostrophecms-page-id-publish) | Publish the draft version of a page | TRUE |
+
+<!-- TODOcument -->
+<!-- |GET | [`/api/v1/@apostrophecms/page/:id?locales=1`](#get-api-v1-apostrophecms-page-id-locales-1) | Request the available locales for a specific page | TRUE |
+|POST | [`/api/v1/@apostrophecms/page/:id/export`](#post-api-v1-apostrophecms-page-id-export) | Copy a page from one locale to another | TRUE |
+|POST | [`/api/v1/@apostrophecms/page/:id/revert-draft-to-published`](#post-api-v1-apostrophecms-page-id-revert-draft-to-published) | Revert a draft page to the the state of the published version, if available | TRUE |
+|POST | [`/api/v1/@apostrophecms/page/:id/revert-published-to-previous`](#post-api-v1-apostrophecms-page-id-revert-published-to-previous) | Revert a published page to the previous version, if available | TRUE | -->
+
```

Comment:
Since we are covering `revert-published-to-previous` we should also cover the new `unpublish` route, which deletes the published mode while leaving the draft mode in place for that document. It may not be used if any descendant pages are still published.